docs/download_files.md

```diff
@@ -0,0 +1,69 @@
+# Downloadable files
+
+The ScPCA portal download packages include gene expression data, a QC report, and associated metadata for each processed sample.
+These files are delivered as a zip file.
+The folder structure within the zip file is determined by selecting to download all samples associated with a project or selecting individual samples.  
+
+## Download folder structure for project downloads: 
+_add in illustration of file structure for project_ 
+
+## Download folder structure for individual sample downloads: 
+_add in illustration of file structure for sample downloads_ 
+
+Sample folders (indicated by the `SCPCS` prefix) contain the files for all libraries (`SCPCL` prefix) derived from that biological sample. 
+Most samples only have one library that has been sequenced. 
+See {ref}`the difference between samples and libraries <faq:what is the difference between a sample and a library?>` for more information.
+
+The files associated with each library are:
+- An unfiltered counts file: `SCPCL000000_unfiltered.rds`, 
+- A filtered counts file: `SCPCL000000_filtered.rds`, 
+- A quality control report: `SCPCL000000_qc.html`, 
+- A metadata file: `SCPCL000000_metadata.json`. 
+Every download also includes a `libraries_metadata.csv` file containing metadata associated with each library included in the download.
+
+## Gene expression data
+
+Gene expression data is available in two forms - as an unfiltered counts file or filtered counts file.
+
+The unfiltered counts file, `SCPCL000000_unfiltered.rds`, is delivered as a RDS file containing a [`SingleCellExperiment` object](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html).
+Within the `SingleCellExperiment` object is the counts matrix, where the rows correspond to genes or features and the columns correspond to cell barcodes. 
+Here, all potential cell barcodes that are identified after running {ref}`Alevin-fry <processing_information:alignment and quantification using alevin-fry>` are included in the counts matrix. 
+Summary statistics for each cell and gene can be found in the `colData` and `rowData` slots of the `SingleCellExperiment` object, respectively.
+The `SingleCellExperiment` object also contains metadata about that particular library, including the versions of Salmon and Alevin-fry used for pre-processing, information about the index used for transcriptome alignment, and parameters used for Alevin-fry. 
```

Comment:
I am not sure how much detail we want to put into this, since we will cover it in a separate section. I would definitely cut down the second sentence, as I am suggesting here. I'm also removing the references to `colData` and `rowData`, but I feel less strongly about that. 

```suggestion
The object also includes summary statistics for each cell barcode and gene, as well as metadata about that particular library, such as the reference index and software versions used for mapping and quantification. 
```